backend/inspirehep/records/config.py

```diff
@@ -118,6 +120,18 @@
                 "completion": {"field": "bookautocomplete"},
             },
         },
+        "record_serializers_aliases": {
+            "bibtex": "application/x-bibtex",
+            "latex_eu": "application/vnd+inspire.latex.eu+x-latex",
+            "latex_us": "application/vnd+inspire.latex.us+x-latex",
```

Comment:
```suggestion
            "latex-eu": "application/vnd+inspire.latex.eu+x-latex",
            "latex-us": "application/vnd+inspire.latex.us+x-latex",
```

Reply:
:+1:  but then we have error in task requirements ;)